### DISCUSSION TITLE
how to access relay

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2420] [Help with using Postal] [answered]
hello, please what is the command to access smtp rrelay in postal, thanks

**Comment by slobodangacesa** [comment, marked as answer]
What do you mean by 'access smtp relay'? Do you want postal to send your email through relay server?
If so, here it is: https://github.com/postalserver/postal/blob/main/config/postal.defaults.yml#L79-L83

**Reply by Ascar699** [reply]
Hi @slobodangacesa I would like to set up a relay so all mail originating from my instance of Postal gets relayed through the likes of MailGun or Sendgrid.  Do you have more details about how to enter SMTP relay credentials like username/password and connection encryption settings?  In the link you shared above we have only hostname and port.  I tried writing my own config and it does not work.  Postal says '550 5.7.1 Relaying denied' and I guess it happens on my side without ever leaving Postal and touching MailGun since I don't see any rejected messages in MailGun logs.

smtp_relays:
  -
    hostname: smtp.mailgun.org
    port: 587
    authentication: LOGIN
    enable_starttls_auto: true
    ssl_mode: Auto
    username: username@mx.mydomain.com
    password: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Appreciate your insights in advance

**Reply by slobodangacesa** [reply]
Since smtp_relay option works only without authentication, you'll need another relaying smtp server, like postfix, that will receive anything from postal ant then authenticate against mailgun, and send them mails.
I guess something like this just might work: https://hub.docker.com/r/juanluisbaptiste/postfix/

**Reply by Ascar699** [reply]
Thanks Slobodan, exactly what I suspected, I somehow assumed that there is no such thing as an option to use relay credentials in Postal and I figured that I will just set up a relay on my iRedMail host which in turn will authenticate against MailGun.

Appreciate your feedback

**Comment by Itseric100** [comment]
thank you so much
